recipes/correct-ts-specifiers/src/fexists.test.ts

```diff
@@ -0,0 +1,135 @@
+import assert from 'node:assert/strict';
+import { type Mock, afterEach, before, describe, it, mock } from 'node:test';
+import { fileURLToPath } from 'node:url';
```

Comment:
```suggestion
import assert from 'node:assert/strict';
import { afterEach, before, describe, it, mock } from 'node:test';
import { fileURLToPath } from 'node:url';
import type { Mock } from 'node:test'
```

I recommend separating js import and type import

Reply:
Not opposed, but why?

Reply:
>  but why?

So it's a bit stupid, but on the one hand you have an import of code (js) that will be executed and on the other hand the typing (ts/d.ts) that will be removed by the transpiler.

Reply:
Hmm, I think combining them is better because it's easier to grok and realise A and type B are coming from the same place. Also, separating them rather significantly increases size.

Reply:
That's true. I think they're both equally good. We'll have to see what the rest of the team thinks.